pkg/watermark/otbucket/otbucket.go

```diff
@@ -0,0 +1,33 @@
+package otbucket
+
+import (
+	"bytes"
+	"encoding/binary"
+)
+
+// OTValue is used in the JetStream offset timeline bucket as the value for the given processor entity key.
+type OTValue struct {
+	Offset    int64
+	Watermark int64
+}
+
+// EncodeToBytes encodes a OTValue object into byte array.
+func (v OTValue) EncodeToBytes() ([]byte, error) {
```

Comment:
Shall we add a simple unit test to encode then decode a OTValue and compare result?

Reply:
That’s why it’s still a draft PR.